unified-runtime/source/adapters/level_zero/v2/command_buffer.cpp

```diff
@@ -483,4 +494,14 @@ urCommandBufferGetInfoExp(ur_exp_command_buffer_handle_t hCommandBuffer,
   return exceptionToResult(std::current_exception());
 }
 
+ur_result_t urCommandBufferEnqueueExp(
+    ur_exp_command_buffer_handle_t CommandBuffer, ur_queue_handle_t UrQueue,
+    uint32_t NumEventsInWaitList, const ur_event_handle_t *EventWaitList,
+    ur_event_handle_t *Event) try {
+  return UrQueue->get().enqueueCommandBufferExp(
+      CommandBuffer, NumEventsInWaitList, EventWaitList, Event);
+} catch (...) {
+  return exceptionToResult(std::current_exception());
+}
+
```

Comment:
Why was this function reintroduced?  I think it might be a rebase error

Reply:
whoops, let me know if i should revert this